demo/src/test/java/com/example/test/tests/KBBMOOCProject/KbbBaseTest.java

```diff
@@ -11,7 +11,7 @@ public class KbbBaseTest {
     
     public WebDriver driver;
     public CustomKeyword keyword;
-    public String pathChromeDriver = "src\\test\\java\\com\\example\\test\\core\\drivers\\driver\\chromedriver.exe";
+    public String pathChromeDriver = "src//test//java//com//example//test//resources//driver//chromedriver";
```

Comment:
it's should revert

Reply:
Have changed to be suitable for window user